docs/buidl.gradle

```diff
@@ -0,0 +1,9 @@
+clean {
+    delete('api/')
+    doLast {
+        exec {
+            workingDir = file('exmaples')
```

Comment:
Shouldn't this produce an error as that directory doesn't exist?

Reply:
Probably because the file is named `buidl.gradle` nothing happens at all?
A little testing of your PR would be good...

Reply:
I have tested `./gradlew nullnessExtraTests` and `./gradlew commandLineTests` with `./gradlew :checker:clean  `, ` ./gradlew loaderTests ` with `./gradlew :framework:clean` and `./gradlew exampleTests` with `./gradlew :docs:clean`